tests/letstest/apache2_targets.yaml

```diff
@@ -33,6 +33,12 @@ targets:
     type: ubuntu
     virt: hvm
     user: admin
+  - ami: ami-0ed54dd1b25657636
+    name: debian9_arm64
+    type: ubuntu
+    virt: hvm
+    user: admin
+    machine_type: a1.medium
```

Comment:
What is the meaning of machine_type?

Reply:
It's the instance type: https://aws.amazon.com/ec2/instance-types/

Maybe `instance_type` is a better name?

Reply:
Yes it is a good name.

Also, so you want to toggle two different types of A1 instances depending on the `--fast` flag?

Reply:
I can change the name.

We could make a separate instance type for the fast flag, but it'd require farther reaching changes. My thought was to skip it until a need comes up since a1.medium is in between t2.micro and t2.medium and seems fast enough.

Reply:
Seems fare to me.